lib/extension.js

```diff
@@ -1,11 +1,11 @@
 const path = require('path');
 
-const EXTRE = /^[.]?[^.]+([.].*)$/;
+const EXTRE = /(\.[^.]+)+$/;
```

Comment:
I will get on the first and last one. For example: I expect 'testing.tmp.babel.js' to output the following possible extensions:
  * .tmp.babel.js
  * .js

That can be one problem. Do you see others?